src/pages/ShopDetail/RestaurantReview/RestaurantReview.scss

```
@@ -0,0 +1,64 @@
+.review {
+  display: flex;
+  padding-top: 50px;
+  padding-bottom: 50px;
```

margin, padding 등 좌표 정보와 함께 사용 가능한 속성은 아래처럼 한 줄로 정의 가능합니다!

```suggestion
  padding: 50px 0;
```
[공식문서](https://developer.mozilla.org/ko/docs/Web/CSS/padding) 참고해주세요!

---

수정했습니다 !